docs/management/debugging.md

```diff
@@ -134,13 +134,13 @@ simple as using the **bt** command:
 
     (gdb) bt
     #0  debugCommand (c=0x7ffc32005000) at debug.c:220
-    #1  0x000000010d246d63 in call (c=0x7ffc32005000) at redis.c:1163
-    #2  0x000000010d247290 in processCommand (c=0x7ffc32005000) at redis.c:1305
+    #1  0x000000010d246d63 in call (c=0x7ffc32005000) at valkey.c:1163
```

Comment:
There is no file called valkey.c. Also in redis there is no redis.c. This text must be very very old.... (We can fix it another time)